executor/runcexecutor/executor_linux.go

```diff
@@ -0,0 +1,308 @@
+package runcexecutor
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"sync"
+	"time"
+
+	"github.com/containerd/console"
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cio"
+	runcV2 "github.com/containerd/containerd/runtime/v2/runc"
```

Comment:
Looking to keep these dependencies somewhat under control. iiuc this is what adds most of them while we only use single `stdio.Platform` helper implementation from there. I wouldn't mind copy/paste for this and eventually refactoring this code so it can be cleanly imported.

Reply:
There are a bunch of deps coming from the reaper logic as well.  I will look into extracting the logic we need for the console and the reaper to see if there is a way to trim what is being pulled in.